Library/Homebrew/extend/os/linux/hardware/cpu.rb

```diff
@@ -107,7 +112,7 @@ def features
       end
 
       def bits
-        Utils.popen_read("getconf", "LONG_BIT").chomp.to_i
+        @bits ||= Utils.popen_read("getconf", "LONG_BIT").chomp.to_i
```

Comment:
It caches the result so that `getconf` isn't executed every time that `bits` is called.